From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Sogup committed:
Update backup-azure-upgrade-backup-to-recovery-services.md

rloutlaw committed:
initial draft of functions content

rloutlaw committed:
add screenshots and clean up

rloutlaw committed:
more clean up

rloutlaw committed:
acrolinx fixes

rloutlaw committed:
clean up images

rloutlaw committed:
add hello azure image

rloutlaw committed:
clean up image

rloutlaw committed:
better sized image

rloutlaw committed:
tightened up next steps a bit

rloutlaw committed:
slight clean up

rloutlaw committed:
fleshing out functions programming guide based on feedback

rloutlaw committed:
minor twekas

rloutlaw committed:
wording changes

rloutlaw committed:
cleaning up functions reference w/ feedback from dev

rloutlaw committed:
toc and image updates

rloutlaw committed:
more functions fixes

rloutlaw committed:
fix for clarity

rloutlaw committed:
more cleanup

rloutlaw committed:
backstop for problems installing core tools 2.0

rloutlaw committed:
clean up the vscode section

rloutlaw committed:
correct some validation warnings related to note blocks

rloutlaw committed:
add java to drop-down selector for each dev reference

sanjeev3 committed:
Add files via upload

sanjeev3 committed:
Update facebook-post-moderation.md

sanjeev3 committed:
Update facebook-post-moderation.md

sanjeev3 committed:
Update facebook-post-moderation.md

sanjeev3 committed:
Update facebook-post-moderation.md

guperrot committed:
Update RUM Android SDK version

rloutlaw committed:
JDKupdated with guidance from Junyi

puicchan committed:
az webapp create link is broken

Should point to https://docs.microsoft.com/en-us/cli/azure/webapp?view=azure-cli-latest#az_webapp_create instead

ggailey777 committed:
some updates to Java

Commit:
Item 675454 edits

rloutlaw committed:
add language selector include

alkohli committed:
created a draft ARM based cmdlets

rloutlaw committed:
fix relative links in include

AnoopVasudavan committed:
fixng unregiste docments

AnoopVasudavan committed:
fixng unregiste docments

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

AnoopVasudavan committed:
fixng unregiste docments

Commit:
fix github repo name and directory pathing

AnoopVasudavan committed:
fixng unregiste docments

gatneil committed:
added TLDR of host vs. guest

nitinme committed:
freshness updates

Commit:
Added AzureAD Get Started walkthrough

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

Commit:
Updated links and grammar fixes.

Commit:
freshness

Commit:
Fixing typo

Commit:
Updated opening line and learn more links heading.

nitinme committed:
fixing acrolinx score

Commit:
formatting

Commit:
Removed redundant 'paid'

gatneil committed:
added CLI commands for SKU api

gatneil committed:
added reference to sku api

linya9191 committed:
Update scenario-churn-prediction.md

Commit:
Fixed last updated date

gatneil committed:
formatting

alkohli committed:
fine tuning 1

v-thepet committed:
fix

alkohli committed:
fine tuning 2

alkohli committed:
fine tuning 3

alkohli committed:
fine tuning 4

alkohli committed:
fine tuning 5

HJToland3 committed:
tiny tweaks

tfitzmac committed:
added full parameter

alkohli committed:
updated TOC to reflect the new doc

alkohli committed:
fine tuning 7

Commit:
Merge pull request #24679 from v-thepet/indexlink

production bug: fix link

PRMerger13 committed:
Merge pull request #24686 from tfitzmac/1003full

added full parameter

nitinme committed:
fixing typo

alkohli committed:
fine tuning 8

rloutlaw committed:
easier to consume next steps

sethmanheim committed:
Adding article on message counters

sethmanheim committed:
Adding to TOC

rloutlaw committed:
quick acrolinx fix

rloutlaw committed:
clipped bash from code fences where any cli would work. Avoid azurecli-interactive because it'll break if you actually do it in the cloud shell

Lisaco88 committed:
Merge pull request #24673 from linya9191/patch-11

Update scenario-churn-prediction.md

PRMerger14 committed:
Merge pull request #24658 from gatneil/description

added TLDR of host vs. guest

Commit:
Merge pull request #24588 from puicchan/patch-3

az webapp create link is broken

PRMerger10 committed:
Merge pull request #24666 from Blackmist/fresh5

freshness

alkohli committed:
fine tuning 8

alkohli committed:
renamed article

cherylmc committed:
table

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into changetbl

nberdy committed:
Update concepts-service.md

GitHubber17 committed:
Merge pull request #24663 from nitinme/nitinme-freshness-updates

Freshness updates - 10/03

Commit:
Minor edits

PRMerger16 committed:
Merge pull request #24701 from nberdy/patch-2

Update concepts-service.md

Lisaco88 committed:
Merge pull request #24511 from sanjeev3/master

Updated the Facebook tutorial with new graphic.

PRMerger17 committed:
Merge pull request #24653 from suhuruli/master

fix github repo name and directory pathing

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into changetbl

v-thepet committed:
make myself author

v-thepet committed:
delete file

cherylmc committed:
update

alkohli committed:
fixed broken link

cherylmc committed:
update

PRMerger14 committed:
Merge pull request #24685 from HJToland3/localworking

tiny tweaks

Commit:
Merge pull request #24608 from iRaindrop/vbruham_oozie

HDInsight - item 675454 fixes for Oozie coordinator time

alkohli committed:
fixed formatting1

tamram committed:
update for public PR #2769

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into changetbl

Commit:
Merge pull request #24668 from mani-ramaswamy/master

Fixing typo

Commit:
Merge pull request #24670 from gatneil/skuApi

Added details for SKU API

PRMerger16 committed:
Merge pull request #24710 from tamram/tamram-1003

update for public PR #2769

AnoopVasudavan committed:
fixing feedback

AnoopVasudavan committed:
Delete vault

AnoopVasudavan committed:
Fixing typo

AnoopVasudavan committed:
Fixing typo

pvrk committed:
Update backup-rbac-rs-vault.md

Modified RBAC requirements

ggailey777 committed:
Changed the Cosmos DB link to beta 4 for Trigger support

PRMerger15 committed:
Merge pull request #23001 from Sogup/patch-7

Note update

PRMerger12 committed:
Merge pull request #24719 from ggailey777/patch-3

Changed the Cosmos DB link to beta 4 for Trigger support

Sogup committed:
Update backup-azure-upgrade-backup-to-recovery-services.md

PRMerger13 committed:
Merge pull request #24720 from Sogup/patch-12

Update backup-azure-upgrade-backup-to-recovery-services.md

PRMerger13 committed:
Merge pull request #24717 from pvrk/patch-19

Update backup-rbac-rs-vault.md

czeumault committed:
Merge pull request #23539 from rloutlaw/java-functions-tutorial

Azure Functions for Java quickstart and developer guide

AnithaAdusumilli committed:
Added a note on public peering

Commit:
freshness

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into fresh5

PRMerger18 committed:
Merge pull request #24729 from Blackmist/fresh5

Freshness

rloutlaw committed:
typo fixing

Commit:
Removed summary step bullets per PR reviewer feedback

PRMerger19 committed:
Merge pull request #24516 from guperrot/rum_update_android_sdk_version

Update RUM Android SDK version

Lisaco88 committed:
Merge pull request #24706 from v-thepet/indexlink

production bug: delete file to clear validation warning

BryanLa committed:
Clarify prereq

Lisaco88 committed:
Merge pull request #24692 from sethmanheim/cv12

Adding article on message counters

PRMerger18 committed:
Merge pull request #24732 from BryanLa/patch-1

Clarify prereq

itechedit committed:
Merge pull request #24731 from rloutlaw/redirect-functions-typo

Add redirect for typo in link from portal

v-shils committed:
Merge pull request #24647 from AnoopVasudavan/master

fixng unregiste docments

christiankuhtz committed:
tweaks

inserted notice about up to one hour delay for AFEC registration effective, couple of minor tweaks

sethmanheim committed:
A couple of minor fixes

alkohli committed:
Feedback from PR approvers

PRMerger19 committed:
Merge pull request #24727 from AnithaAdusumilli/patch-3

Added a note on public peering

PRMerger15 committed:
Merge pull request #24736 from sethmanheim/wk10-4

A couple of minor fixes

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Commit:
Updated required metadata

PRMerger17 committed:
Merge pull request #24735 from christiankuhtz/patch-11

tweaks

v-alje committed:
Merge pull request #24737 from MicrosoftDocs/FromPublicRepo

From public repo

itechedit committed:
Merge pull request #24699 from cherylmc/changetbl

Changetbl

v-shils committed:
Merge pull request #24661 from jeffgilb/jeff104

Added AzureAD Get Started walkthrough

Lisaco88 committed:
Merge pull request #24676 from alkohli/ssarm

Go live 10/5 @ 10AM or before - About StorSimple Azure Resource Manager SDK-based scripts
